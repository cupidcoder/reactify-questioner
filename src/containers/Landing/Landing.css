.LandingPage {
  min-height: 100vh;
  background: linear-gradient(rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.6)), url(../../assets//images/bg-hero.jpg);
  width: 100%;
  background-size: cover;
  background-position: center center;
  background-repeat: no-repeat;
}

.HeroText {
  width: 70%;
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
}

.HeroText h1 {
  text-align: center;
  color: #ffffff;
  font-size: 3rem;
}

.Button {
  width: 10rem;
  margin: 2.5rem auto;
}
