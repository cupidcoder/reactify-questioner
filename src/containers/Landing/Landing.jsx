import React from 'react';
import { Link } from 'react-router-dom';
import Button from '../../components/Button/Button';
import style from './Landing.css';

const LandingPage = () => (
  <section className={style.LandingPage}>
    <div className={style.HeroText}>
      <h1>Don&rsquo;t leave your attendees guessing</h1>
      <div className={style.Button}>
        <Button>
          <Link to='/signup'>
            Sign up
          </Link>
        </Button>
      </div>
    </div>
  </section>
);

export default LandingPage;
