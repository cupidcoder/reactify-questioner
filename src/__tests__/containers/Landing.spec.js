import React from 'react';
import { shallow } from 'enzyme';
import Landing from '../../containers/Landing/Landing';

describe('Landing', () => {
  it('Should correctly save snapshot of the Landing', () => {
    const wrapper = shallow(<Landing />);
    expect(wrapper).toMatchSnapshot();
  });
});
