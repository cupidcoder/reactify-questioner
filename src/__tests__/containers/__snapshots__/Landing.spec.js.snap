// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Should correctly save snapshot of the Landing 1`] = `ShallowWrapper {}`;
